pom.xml

```diff
@@ -174,6 +174,11 @@
                 <artifactId>jackson-databind</artifactId>
                 <version>${jackson.databind.version}</version>
             </dependency>
+            <dependency>
+                <groupId>com.google.protobuf</groupId>
+                <artifactId>protobuf-java</artifactId>
+                <version>${protobuf.version}</version>
```

Comment:
Nice!  Probably, if necessary, should also do protobuf-java-util since some places import that one too and might as well keep them the same version.

Reply:
good catch. updated